docs/troubleshooting/report-scylla-problem.rst

```diff
@@ -6,7 +6,7 @@ In the event there is an issue you would like to report to ScyllaDB support, you
 
 In general, there are two types of issues:
 
-* **Scylla failure** - There is some kind of failure, possibly due to a connectivity issue, a timeout, or otherwise, where the Scylla server or the Scylla nodes are not working. These cases require you to send ScyllaDB support both a `Health Check Report`_ as well as `Core Dump`_ files (if available).
+* **Scylla failure** - There is some kind of failure, possibly due to a connectivity issue, a timeout, or otherwise, where the Scylla server or the Scylla nodes are not working. These cases require you to send ScyllaDB support both :ref:`Scylla Doctor Vitals <report-scylla-problem-scylla-doctor>` and `Core Dump`_ files (if available).
```

Comment:
And scylla logs (collected by Scylla Doctor into a separate file). Please see https://cloudsupportscylla.zendesk.com/hc/en-us/articles/15910490082845-How-to-collect-Scylla-Doctor-vitals for more details.

Reply:
@vladzcloudius the instruction in that link doesn't cover how to configure it in case CQL is TLS encrypted, there a way todo that ?

Reply:
I wasn't able to find ScyllaDB cqlsh related docs on the matter (similar to this one: https://docs.datastax.com/en/cassandra-oss/3.0/cassandra/configuration/secureCqlshSSL.html). 

@annastuchlik could you, please, tell me if it hides somewhere?

Reply:
> Even so you might have ssl configuration, one still needs to add --ssl to the command-line for it to work, I'm not sure cqlsh has a way to force ssl without it.

A more standard way is to configure it via `.cqlshrc`